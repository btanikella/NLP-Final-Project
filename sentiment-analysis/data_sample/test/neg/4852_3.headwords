kept
stumbles
leaves
plays
singer
call
take
comes
key
felt
