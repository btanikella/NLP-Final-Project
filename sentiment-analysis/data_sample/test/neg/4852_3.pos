PRP VBD VBG IN DT NN TO VB  CC  RB  DT VBZ DT JJS NN IN JJ NN : DT JJ NN WDT VBZ IN DT NN IN PRP$ JJ JJ NN
DT NN VBZ IN IN DT NN VBN IN DT NNP
DT NN IN JJ NN VBZ DT JJ NNS IN JJ TO VB JJ IN VB JJ IN JJ NN
NNS VBZ RB DT JJ NN WP VBZ VBD DT NN IN DT NN
NNP VBZ DT VBG NN CC RB JJ IN DT NN
RB  DT NN VBZ RB RB VB IN PRP TO VB RB IN DT NN NN NN
PDT DT RB VBN  RB  JJ NNS VBP VBZ VBG DT NN IN DT NN IN JJ NN
DT JJ NN VBZ IN IN DT RB JJ NNP NNP NN
IN PRP VBP JJ CD CC VB NNP VBZ DT JJ TO NN  PRP MD RB NN DT NN IN NN IN DT NN
RB  IN PRP VBD DT NN PRP RB VBD JJ IN VBG NNP POS JJ NN IN NNP  CC PRP MD RB VB VBN IN NNP NNP IN VBG
