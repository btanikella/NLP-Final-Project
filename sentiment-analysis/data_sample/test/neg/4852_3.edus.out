Sentence #1 (36 tokens):
I kept waiting for this film to improve, but, alack, this is the worst kind of escapist movie: a spun-sugar confection that sinks under the weight of its own ponderous self-importance.
[Text=I CharacterOffsetBegin=4 CharacterOffsetEnd=5 PartOfSpeech=PRP] [Text=kept CharacterOffsetBegin=6 CharacterOffsetEnd=10 PartOfSpeech=VBD] [Text=waiting CharacterOffsetBegin=11 CharacterOffsetEnd=18 PartOfSpeech=VBG] [Text=for CharacterOffsetBegin=19 CharacterOffsetEnd=22 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=23 CharacterOffsetEnd=27 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=28 CharacterOffsetEnd=32 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=33 CharacterOffsetEnd=35 PartOfSpeech=TO] [Text=improve CharacterOffsetBegin=36 CharacterOffsetEnd=43 PartOfSpeech=VB] [Text=, CharacterOffsetBegin=43 CharacterOffsetEnd=44 PartOfSpeech=,] [Text=but CharacterOffsetBegin=45 CharacterOffsetEnd=48 PartOfSpeech=CC] [Text=, CharacterOffsetBegin=48 CharacterOffsetEnd=49 PartOfSpeech=,] [Text=alack CharacterOffsetBegin=50 CharacterOffsetEnd=55 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=55 CharacterOffsetEnd=56 PartOfSpeech=,] [Text=this CharacterOffsetBegin=57 CharacterOffsetEnd=61 PartOfSpeech=DT] [Text=is CharacterOffsetBegin=62 CharacterOffsetEnd=64 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=65 CharacterOffsetEnd=68 PartOfSpeech=DT] [Text=worst CharacterOffsetBegin=69 CharacterOffsetEnd=74 PartOfSpeech=JJS] [Text=kind CharacterOffsetBegin=75 CharacterOffsetEnd=79 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=80 CharacterOffsetEnd=82 PartOfSpeech=IN] [Text=escapist CharacterOffsetBegin=83 CharacterOffsetEnd=91 PartOfSpeech=JJ] [Text=movie CharacterOffsetBegin=92 CharacterOffsetEnd=97 PartOfSpeech=NN] [Text=: CharacterOffsetBegin=97 CharacterOffsetEnd=98 PartOfSpeech=:] [Text=a CharacterOffsetBegin=99 CharacterOffsetEnd=100 PartOfSpeech=DT] [Text=spun-sugar CharacterOffsetBegin=101 CharacterOffsetEnd=111 PartOfSpeech=JJ] [Text=confection CharacterOffsetBegin=112 CharacterOffsetEnd=122 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=123 CharacterOffsetEnd=127 PartOfSpeech=WDT] [Text=sinks CharacterOffsetBegin=128 CharacterOffsetEnd=133 PartOfSpeech=VBZ] [Text=under CharacterOffsetBegin=134 CharacterOffsetEnd=139 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=140 CharacterOffsetEnd=143 PartOfSpeech=DT] [Text=weight CharacterOffsetBegin=144 CharacterOffsetEnd=150 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=151 CharacterOffsetEnd=153 PartOfSpeech=IN] [Text=its CharacterOffsetBegin=154 CharacterOffsetEnd=157 PartOfSpeech=PRP$] [Text=own CharacterOffsetBegin=158 CharacterOffsetEnd=161 PartOfSpeech=JJ] [Text=ponderous CharacterOffsetBegin=162 CharacterOffsetEnd=171 PartOfSpeech=JJ] [Text=self-importance CharacterOffsetBegin=172 CharacterOffsetEnd=187 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=187 CharacterOffsetEnd=188 PartOfSpeech=.] 
root(ROOT-0, kept-2)
nsubj(kept-2, I-1)
xcomp(kept-2, waiting-3)
mark(improve-8, for-4)
det(film-6, this-5)
nsubj(improve-8, film-6)
aux(improve-8, to-7)
advcl(waiting-3, improve-8)
punct(improve-8, ,-9)
cc(improve-8, but-10)
punct(improve-8, ,-11)
advmod(kind-18, alack-12)
punct(kind-18, ,-13)
nsubj(kind-18, this-14)
cop(kind-18, is-15)
det(kind-18, the-16)
amod(kind-18, worst-17)
conj(improve-8, kind-18)
prep(kind-18, of-19)
amod(movie-21, escapist-20)
pobj(of-19, movie-21)
punct(kind-18, :-22)
det(confection-25, a-23)
amod(confection-25, spun-sugar-24)
dep(kind-18, confection-25)
nsubj(sinks-27, that-26)
rcmod(confection-25, sinks-27)
prep(sinks-27, under-28)
det(weight-30, the-29)
pobj(under-28, weight-30)
prep(weight-30, of-31)
poss(self-importance-35, its-32)
amod(self-importance-35, own-33)
amod(self-importance-35, ponderous-34)
pobj(of-31, self-importance-35)
punct(kept-2, .-36)

Sentence #2 (12 tokens):
The pace stumbles on like a legionnaire stranded in the Sahara.
[Text=The CharacterOffsetBegin=193 CharacterOffsetEnd=196 PartOfSpeech=DT] [Text=pace CharacterOffsetBegin=197 CharacterOffsetEnd=201 PartOfSpeech=NN] [Text=stumbles CharacterOffsetBegin=202 CharacterOffsetEnd=210 PartOfSpeech=VBZ] [Text=on CharacterOffsetBegin=211 CharacterOffsetEnd=213 PartOfSpeech=IN] [Text=like CharacterOffsetBegin=214 CharacterOffsetEnd=218 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=219 CharacterOffsetEnd=220 PartOfSpeech=DT] [Text=legionnaire CharacterOffsetBegin=221 CharacterOffsetEnd=232 PartOfSpeech=NN] [Text=stranded CharacterOffsetBegin=233 CharacterOffsetEnd=241 PartOfSpeech=VBN] [Text=in CharacterOffsetBegin=242 CharacterOffsetEnd=244 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=245 CharacterOffsetEnd=248 PartOfSpeech=DT] [Text=Sahara CharacterOffsetBegin=249 CharacterOffsetEnd=255 PartOfSpeech=NNP] [Text=. CharacterOffsetBegin=255 CharacterOffsetEnd=256 PartOfSpeech=.] 
root(ROOT-0, stumbles-3)
det(pace-2, The-1)
nsubj(stumbles-3, pace-2)
prep(stumbles-3, on-4)
pcomp(on-4, like-5)
det(legionnaire-7, a-6)
pobj(like-5, legionnaire-7)
partmod(legionnaire-7, stranded-8)
prep(stranded-8, in-9)
det(Sahara-11, the-10)
pobj(in-9, Sahara-11)
punct(stumbles-3, .-12)

Sentence #3 (21 tokens):
The absence of good dialogue leaves the appealing stars with little to do other than look good in white linen.
[Text=The CharacterOffsetBegin=261 CharacterOffsetEnd=264 PartOfSpeech=DT] [Text=absence CharacterOffsetBegin=265 CharacterOffsetEnd=272 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=273 CharacterOffsetEnd=275 PartOfSpeech=IN] [Text=good CharacterOffsetBegin=276 CharacterOffsetEnd=280 PartOfSpeech=JJ] [Text=dialogue CharacterOffsetBegin=281 CharacterOffsetEnd=289 PartOfSpeech=NN] [Text=leaves CharacterOffsetBegin=290 CharacterOffsetEnd=296 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=297 CharacterOffsetEnd=300 PartOfSpeech=DT] [Text=appealing CharacterOffsetBegin=301 CharacterOffsetEnd=310 PartOfSpeech=JJ] [Text=stars CharacterOffsetBegin=311 CharacterOffsetEnd=316 PartOfSpeech=NNS] [Text=with CharacterOffsetBegin=317 CharacterOffsetEnd=321 PartOfSpeech=IN] [Text=little CharacterOffsetBegin=322 CharacterOffsetEnd=328 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=329 CharacterOffsetEnd=331 PartOfSpeech=TO] [Text=do CharacterOffsetBegin=332 CharacterOffsetEnd=334 PartOfSpeech=VB] [Text=other CharacterOffsetBegin=335 CharacterOffsetEnd=340 PartOfSpeech=JJ] [Text=than CharacterOffsetBegin=341 CharacterOffsetEnd=345 PartOfSpeech=IN] [Text=look CharacterOffsetBegin=346 CharacterOffsetEnd=350 PartOfSpeech=VB] [Text=good CharacterOffsetBegin=351 CharacterOffsetEnd=355 PartOfSpeech=JJ] [Text=in CharacterOffsetBegin=356 CharacterOffsetEnd=358 PartOfSpeech=IN] [Text=white CharacterOffsetBegin=359 CharacterOffsetEnd=364 PartOfSpeech=JJ] [Text=linen CharacterOffsetBegin=365 CharacterOffsetEnd=370 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=370 CharacterOffsetEnd=371 PartOfSpeech=.] 
root(ROOT-0, leaves-6)
det(absence-2, The-1)
nsubj(leaves-6, absence-2)
prep(absence-2, of-3)
amod(dialogue-5, good-4)
pobj(of-3, dialogue-5)
det(stars-9, the-7)
amod(stars-9, appealing-8)
dobj(leaves-6, stars-9)
prep(leaves-6, with-10)
pobj(with-10, little-11)
aux(do-13, to-12)
infmod(little-11, do-13)
dobj(do-13, other-14)
mark(look-16, than-15)
dep(other-14, look-16)
acomp(look-16, good-17)
prep(look-16, in-18)
amod(linen-20, white-19)
pobj(in-18, linen-20)
punct(leaves-6, .-21)

Sentence #4 (15 tokens):
Irons plays yet another moneyed charmer who's had a touch of the sun.
[Text=Irons CharacterOffsetBegin=376 CharacterOffsetEnd=381 PartOfSpeech=NNS] [Text=plays CharacterOffsetBegin=382 CharacterOffsetEnd=387 PartOfSpeech=VBZ] [Text=yet CharacterOffsetBegin=388 CharacterOffsetEnd=391 PartOfSpeech=RB] [Text=another CharacterOffsetBegin=392 CharacterOffsetEnd=399 PartOfSpeech=DT] [Text=moneyed CharacterOffsetBegin=400 CharacterOffsetEnd=407 PartOfSpeech=JJ] [Text=charmer CharacterOffsetBegin=408 CharacterOffsetEnd=415 PartOfSpeech=NN] [Text=who CharacterOffsetBegin=416 CharacterOffsetEnd=419 PartOfSpeech=WP] [Text='s CharacterOffsetBegin=419 CharacterOffsetEnd=421 PartOfSpeech=VBZ] [Text=had CharacterOffsetBegin=422 CharacterOffsetEnd=425 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=426 CharacterOffsetEnd=427 PartOfSpeech=DT] [Text=touch CharacterOffsetBegin=428 CharacterOffsetEnd=433 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=434 CharacterOffsetEnd=436 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=437 CharacterOffsetEnd=440 PartOfSpeech=DT] [Text=sun CharacterOffsetBegin=441 CharacterOffsetEnd=444 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=444 CharacterOffsetEnd=445 PartOfSpeech=.] 
root(ROOT-0, plays-2)
nsubj(plays-2, Irons-1)
advmod(plays-2, yet-3)
det(charmer-6, another-4)
amod(charmer-6, moneyed-5)
dobj(plays-2, charmer-6)
nsubjpass(had-9, who-7)
auxpass(had-9, 's-8)
rcmod(charmer-6, had-9)
det(touch-11, a-10)
dobj(had-9, touch-11)
prep(touch-11, of-12)
det(sun-14, the-13)
pobj(of-12, sun-14)
punct(plays-2, .-15)

Sentence #5 (12 tokens):
Kaas is a pleasing singer but not much of an actress.
[Text=Kaas CharacterOffsetBegin=450 CharacterOffsetEnd=454 PartOfSpeech=NNP] [Text=is CharacterOffsetBegin=455 CharacterOffsetEnd=457 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=458 CharacterOffsetEnd=459 PartOfSpeech=DT] [Text=pleasing CharacterOffsetBegin=460 CharacterOffsetEnd=468 PartOfSpeech=VBG] [Text=singer CharacterOffsetBegin=469 CharacterOffsetEnd=475 PartOfSpeech=NN] [Text=but CharacterOffsetBegin=476 CharacterOffsetEnd=479 PartOfSpeech=CC] [Text=not CharacterOffsetBegin=480 CharacterOffsetEnd=483 PartOfSpeech=RB] [Text=much CharacterOffsetBegin=484 CharacterOffsetEnd=488 PartOfSpeech=JJ] [Text=of CharacterOffsetBegin=489 CharacterOffsetEnd=491 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=492 CharacterOffsetEnd=494 PartOfSpeech=DT] [Text=actress CharacterOffsetBegin=495 CharacterOffsetEnd=502 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=502 CharacterOffsetEnd=503 PartOfSpeech=.] 
root(ROOT-0, singer-5)
nsubj(singer-5, Kaas-1)
cop(singer-5, is-2)
det(singer-5, a-3)
amod(singer-5, pleasing-4)
cc(singer-5, but-6)
neg(much-8, not-7)
conj(singer-5, much-8)
prep(much-8, of-9)
det(actress-11, an-10)
pobj(of-9, actress-11)
punct(singer-5, .-12)

Sentence #6 (19 tokens):
Luckily, the script does not often call on her to emote away from the jazz club microphone.
[Text=Luckily CharacterOffsetBegin=508 CharacterOffsetEnd=515 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=515 CharacterOffsetEnd=516 PartOfSpeech=,] [Text=the CharacterOffsetBegin=517 CharacterOffsetEnd=520 PartOfSpeech=DT] [Text=script CharacterOffsetBegin=521 CharacterOffsetEnd=527 PartOfSpeech=NN] [Text=does CharacterOffsetBegin=528 CharacterOffsetEnd=532 PartOfSpeech=VBZ] [Text=not CharacterOffsetBegin=533 CharacterOffsetEnd=536 PartOfSpeech=RB] [Text=often CharacterOffsetBegin=537 CharacterOffsetEnd=542 PartOfSpeech=RB] [Text=call CharacterOffsetBegin=543 CharacterOffsetEnd=547 PartOfSpeech=VB] [Text=on CharacterOffsetBegin=548 CharacterOffsetEnd=550 PartOfSpeech=IN] [Text=her CharacterOffsetBegin=551 CharacterOffsetEnd=554 PartOfSpeech=PRP] [Text=to CharacterOffsetBegin=555 CharacterOffsetEnd=557 PartOfSpeech=TO] [Text=emote CharacterOffsetBegin=558 CharacterOffsetEnd=563 PartOfSpeech=VB] [Text=away CharacterOffsetBegin=564 CharacterOffsetEnd=568 PartOfSpeech=RB] [Text=from CharacterOffsetBegin=569 CharacterOffsetEnd=573 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=574 CharacterOffsetEnd=577 PartOfSpeech=DT] [Text=jazz CharacterOffsetBegin=578 CharacterOffsetEnd=582 PartOfSpeech=NN] [Text=club CharacterOffsetBegin=583 CharacterOffsetEnd=587 PartOfSpeech=NN] [Text=microphone CharacterOffsetBegin=588 CharacterOffsetEnd=598 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=598 CharacterOffsetEnd=599 PartOfSpeech=.] 
root(ROOT-0, call-8)
advmod(call-8, Luckily-1)
punct(call-8, ,-2)
det(script-4, the-3)
nsubj(call-8, script-4)
aux(call-8, does-5)
neg(call-8, not-6)
advmod(call-8, often-7)
prep(call-8, on-9)
pobj(on-9, her-10)
aux(emote-12, to-11)
xcomp(call-8, emote-12)
advmod(emote-12, away-13)
prep(away-13, from-14)
det(microphone-18, the-15)
nn(microphone-18, jazz-16)
nn(microphone-18, club-17)
pobj(from-14, microphone-18)
punct(call-8, .-19)

Sentence #7 (21 tokens):
All the enviably relaxed, pretty, unnecessary characters take turns masticating the scenery with an air of weary sophistication.
[Text=All CharacterOffsetBegin=604 CharacterOffsetEnd=607 PartOfSpeech=PDT] [Text=the CharacterOffsetBegin=608 CharacterOffsetEnd=611 PartOfSpeech=DT] [Text=enviably CharacterOffsetBegin=612 CharacterOffsetEnd=620 PartOfSpeech=RB] [Text=relaxed CharacterOffsetBegin=621 CharacterOffsetEnd=628 PartOfSpeech=VBN] [Text=, CharacterOffsetBegin=628 CharacterOffsetEnd=629 PartOfSpeech=,] [Text=pretty CharacterOffsetBegin=630 CharacterOffsetEnd=636 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=636 CharacterOffsetEnd=637 PartOfSpeech=,] [Text=unnecessary CharacterOffsetBegin=638 CharacterOffsetEnd=649 PartOfSpeech=JJ] [Text=characters CharacterOffsetBegin=650 CharacterOffsetEnd=660 PartOfSpeech=NNS] [Text=take CharacterOffsetBegin=661 CharacterOffsetEnd=665 PartOfSpeech=VBP] [Text=turns CharacterOffsetBegin=666 CharacterOffsetEnd=671 PartOfSpeech=VBZ] [Text=masticating CharacterOffsetBegin=672 CharacterOffsetEnd=683 PartOfSpeech=VBG] [Text=the CharacterOffsetBegin=684 CharacterOffsetEnd=687 PartOfSpeech=DT] [Text=scenery CharacterOffsetBegin=688 CharacterOffsetEnd=695 PartOfSpeech=NN] [Text=with CharacterOffsetBegin=696 CharacterOffsetEnd=700 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=701 CharacterOffsetEnd=703 PartOfSpeech=DT] [Text=air CharacterOffsetBegin=704 CharacterOffsetEnd=707 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=708 CharacterOffsetEnd=710 PartOfSpeech=IN] [Text=weary CharacterOffsetBegin=711 CharacterOffsetEnd=716 PartOfSpeech=JJ] [Text=sophistication CharacterOffsetBegin=717 CharacterOffsetEnd=731 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=731 CharacterOffsetEnd=732 PartOfSpeech=.] 
root(ROOT-0, take-10)
dep(relaxed-4, All-1)
dep(relaxed-4, the-2)
advmod(relaxed-4, enviably-3)
advcl(take-10, relaxed-4)
punct(relaxed-4, ,-5)
advmod(relaxed-4, pretty-6)
punct(relaxed-4, ,-7)
amod(characters-9, unnecessary-8)
nsubj(take-10, characters-9)
dobj(take-10, turns-11)
xcomp(turns-11, masticating-12)
det(scenery-14, the-13)
dobj(masticating-12, scenery-14)
prep(masticating-12, with-15)
det(air-17, an-16)
pobj(with-15, air-17)
prep(air-17, of-18)
amod(sophistication-20, weary-19)
pobj(of-18, sophistication-20)
punct(take-10, .-21)

Sentence #8 (13 tokens):
The whole exercise comes across like an interminably long Ralph Lauren ad.
[Text=The CharacterOffsetBegin=737 CharacterOffsetEnd=740 PartOfSpeech=DT] [Text=whole CharacterOffsetBegin=741 CharacterOffsetEnd=746 PartOfSpeech=JJ] [Text=exercise CharacterOffsetBegin=747 CharacterOffsetEnd=755 PartOfSpeech=NN] [Text=comes CharacterOffsetBegin=756 CharacterOffsetEnd=761 PartOfSpeech=VBZ] [Text=across CharacterOffsetBegin=762 CharacterOffsetEnd=768 PartOfSpeech=IN] [Text=like CharacterOffsetBegin=769 CharacterOffsetEnd=773 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=774 CharacterOffsetEnd=776 PartOfSpeech=DT] [Text=interminably CharacterOffsetBegin=777 CharacterOffsetEnd=789 PartOfSpeech=RB] [Text=long CharacterOffsetBegin=790 CharacterOffsetEnd=794 PartOfSpeech=JJ] [Text=Ralph CharacterOffsetBegin=795 CharacterOffsetEnd=800 PartOfSpeech=NNP] [Text=Lauren CharacterOffsetBegin=801 CharacterOffsetEnd=807 PartOfSpeech=NNP] [Text=ad CharacterOffsetBegin=808 CharacterOffsetEnd=810 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=810 CharacterOffsetEnd=811 PartOfSpeech=.] 
root(ROOT-0, comes-4)
det(exercise-3, The-1)
amod(exercise-3, whole-2)
nsubj(comes-4, exercise-3)
prep(comes-4, across-5)
pcomp(across-5, like-6)
det(ad-12, an-7)
amod(ad-12, interminably-8)
amod(ad-12, long-9)
nn(ad-12, Ralph-10)
nn(ad-12, Lauren-11)
pobj(like-6, ad-12)
punct(comes-4, .-13)

Sentence #9 (26 tokens):
If you're past forty and believe Francophilia is the key to sophistication, you may well mistake this piece of cardboard for a baguette.
[Text=If CharacterOffsetBegin=816 CharacterOffsetEnd=818 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=819 CharacterOffsetEnd=822 PartOfSpeech=PRP] [Text='re CharacterOffsetBegin=822 CharacterOffsetEnd=825 PartOfSpeech=VBP] [Text=past CharacterOffsetBegin=826 CharacterOffsetEnd=830 PartOfSpeech=JJ] [Text=forty CharacterOffsetBegin=831 CharacterOffsetEnd=836 PartOfSpeech=CD] [Text=and CharacterOffsetBegin=837 CharacterOffsetEnd=840 PartOfSpeech=CC] [Text=believe CharacterOffsetBegin=841 CharacterOffsetEnd=848 PartOfSpeech=VB] [Text=Francophilia CharacterOffsetBegin=849 CharacterOffsetEnd=861 PartOfSpeech=NNP] [Text=is CharacterOffsetBegin=862 CharacterOffsetEnd=864 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=865 CharacterOffsetEnd=868 PartOfSpeech=DT] [Text=key CharacterOffsetBegin=869 CharacterOffsetEnd=872 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=873 CharacterOffsetEnd=875 PartOfSpeech=TO] [Text=sophistication CharacterOffsetBegin=876 CharacterOffsetEnd=890 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=890 CharacterOffsetEnd=891 PartOfSpeech=,] [Text=you CharacterOffsetBegin=892 CharacterOffsetEnd=895 PartOfSpeech=PRP] [Text=may CharacterOffsetBegin=896 CharacterOffsetEnd=899 PartOfSpeech=MD] [Text=well CharacterOffsetBegin=900 CharacterOffsetEnd=904 PartOfSpeech=RB] [Text=mistake CharacterOffsetBegin=905 CharacterOffsetEnd=912 PartOfSpeech=NN] [Text=this CharacterOffsetBegin=913 CharacterOffsetEnd=917 PartOfSpeech=DT] [Text=piece CharacterOffsetBegin=918 CharacterOffsetEnd=923 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=924 CharacterOffsetEnd=926 PartOfSpeech=IN] [Text=cardboard CharacterOffsetBegin=927 CharacterOffsetEnd=936 PartOfSpeech=NN] [Text=for CharacterOffsetBegin=937 CharacterOffsetEnd=940 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=941 CharacterOffsetEnd=942 PartOfSpeech=DT] [Text=baguette CharacterOffsetBegin=943 CharacterOffsetEnd=951 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=951 CharacterOffsetEnd=952 PartOfSpeech=.] 
root(ROOT-0, key-11)
mark(forty-5, If-1)
nsubj(forty-5, you-2)
cop(forty-5, 're-3)
amod(forty-5, past-4)
advcl(key-11, forty-5)
cc(forty-5, and-6)
conj(forty-5, believe-7)
nsubj(key-11, Francophilia-8)
cop(key-11, is-9)
det(key-11, the-10)
prep(key-11, to-12)
pobj(to-12, sophistication-13)
punct(key-11, ,-14)
nsubj(mistake-18, you-15)
aux(mistake-18, may-16)
advmod(mistake-18, well-17)
ccomp(key-11, mistake-18)
det(piece-20, this-19)
dobj(mistake-18, piece-20)
prep(piece-20, of-21)
pobj(of-21, cardboard-22)
prep(mistake-18, for-23)
det(baguette-25, a-24)
pobj(for-23, baguette-25)
punct(key-11, .-26)

Sentence #10 (32 tokens):
Well, if you liked this movie you probably felt smart for appreciating Godard's leaden loge de l'amour, and you may even have sat through Le Divorce without cringing.
[Text=Well CharacterOffsetBegin=957 CharacterOffsetEnd=961 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=961 CharacterOffsetEnd=962 PartOfSpeech=,] [Text=if CharacterOffsetBegin=963 CharacterOffsetEnd=965 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=966 CharacterOffsetEnd=969 PartOfSpeech=PRP] [Text=liked CharacterOffsetBegin=970 CharacterOffsetEnd=975 PartOfSpeech=VBD] [Text=this CharacterOffsetBegin=976 CharacterOffsetEnd=980 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=981 CharacterOffsetEnd=986 PartOfSpeech=NN] [Text=you CharacterOffsetBegin=987 CharacterOffsetEnd=990 PartOfSpeech=PRP] [Text=probably CharacterOffsetBegin=991 CharacterOffsetEnd=999 PartOfSpeech=RB] [Text=felt CharacterOffsetBegin=1000 CharacterOffsetEnd=1004 PartOfSpeech=VBD] [Text=smart CharacterOffsetBegin=1005 CharacterOffsetEnd=1010 PartOfSpeech=JJ] [Text=for CharacterOffsetBegin=1011 CharacterOffsetEnd=1014 PartOfSpeech=IN] [Text=appreciating CharacterOffsetBegin=1015 CharacterOffsetEnd=1027 PartOfSpeech=VBG] [Text=Godard CharacterOffsetBegin=1028 CharacterOffsetEnd=1034 PartOfSpeech=NNP] [Text='s CharacterOffsetBegin=1034 CharacterOffsetEnd=1036 PartOfSpeech=POS] [Text=leaden CharacterOffsetBegin=1037 CharacterOffsetEnd=1043 PartOfSpeech=JJ] [Text=loge CharacterOffsetBegin=1044 CharacterOffsetEnd=1048 PartOfSpeech=NN] [Text=de CharacterOffsetBegin=1049 CharacterOffsetEnd=1051 PartOfSpeech=IN] [Text=l'amour CharacterOffsetBegin=1052 CharacterOffsetEnd=1059 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=1059 CharacterOffsetEnd=1060 PartOfSpeech=,] [Text=and CharacterOffsetBegin=1061 CharacterOffsetEnd=1064 PartOfSpeech=CC] [Text=you CharacterOffsetBegin=1065 CharacterOffsetEnd=1068 PartOfSpeech=PRP] [Text=may CharacterOffsetBegin=1069 CharacterOffsetEnd=1072 PartOfSpeech=MD] [Text=even CharacterOffsetBegin=1073 CharacterOffsetEnd=1077 PartOfSpeech=RB] [Text=have CharacterOffsetBegin=1078 CharacterOffsetEnd=1082 PartOfSpeech=VB] [Text=sat CharacterOffsetBegin=1083 CharacterOffsetEnd=1086 PartOfSpeech=VBN] [Text=through CharacterOffsetBegin=1087 CharacterOffsetEnd=1094 PartOfSpeech=IN] [Text=Le CharacterOffsetBegin=1095 CharacterOffsetEnd=1097 PartOfSpeech=NNP] [Text=Divorce CharacterOffsetBegin=1098 CharacterOffsetEnd=1105 PartOfSpeech=NNP] [Text=without CharacterOffsetBegin=1106 CharacterOffsetEnd=1113 PartOfSpeech=IN] [Text=cringing CharacterOffsetBegin=1114 CharacterOffsetEnd=1122 PartOfSpeech=VBG] [Text=. CharacterOffsetBegin=1122 CharacterOffsetEnd=1123 PartOfSpeech=.] 
root(ROOT-0, felt-10)
advmod(felt-10, Well-1)
punct(felt-10, ,-2)
mark(liked-5, if-3)
nsubj(liked-5, you-4)
advcl(felt-10, liked-5)
det(movie-7, this-6)
tmod(felt-10, movie-7)
nsubj(felt-10, you-8)
advmod(felt-10, probably-9)
acomp(felt-10, smart-11)
prep(felt-10, for-12)
pcomp(for-12, appreciating-13)
poss(l'amour-19, Godard-14)
possessive(Godard-14, 's-15)
amod(l'amour-19, leaden-16)
nn(l'amour-19, loge-17)
nn(l'amour-19, de-18)
dobj(appreciating-13, l'amour-19)
punct(felt-10, ,-20)
cc(felt-10, and-21)
nsubj(sat-26, you-22)
aux(sat-26, may-23)
advmod(sat-26, even-24)
aux(sat-26, have-25)
conj(felt-10, sat-26)
prep(sat-26, through-27)
nn(Divorce-29, Le-28)
pobj(through-27, Divorce-29)
prep(sat-26, without-30)
pcomp(without-30, cringing-31)
punct(felt-10, .-32)

